applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -793,9 +799,24 @@ void audio_datapath_sdu_ref_update(uint32_t sdu_ref_us)
 	}
 }
 
-void audio_datapath_stream_out(const uint8_t *buf, size_t size, uint32_t sdu_ref_us, bool bad_frame,
-			       uint32_t recv_frame_ts_us)
+void audio_datapath_stream_out(const uint8_t *buf, size_t size, uint32_t sdu_ref_us,
+			       uint16_t seq_num, bool bad_frame, uint32_t recv_frame_ts_us)
```

Comment:
As `seq_num` is a `uint16_t` this will wrap every ~minute given a 1 ms conn interval. This needs to be handled.

Reply:
seq_num is currently defined as an uint16_t in "iso.h" in the struct "bt_iso_tx_info".

Reply:
That is fine as long as the wrapping is handled.